pkgs/development/python-modules/publicsuffixlist/default.nix

```
@@ -9,14 +9,14 @@
 
 buildPythonPackage rec {
   pname = "publicsuffixlist";
-  version = "0.10.0.20231026";
+  version = "0.10.0.20231028";
   format = "setuptools";
```

Please start converting setuptools format to PEP517 builds.

```
publicsuffixlist> !!
publicsuffixlist> 
publicsuffixlist>         ********************************************************************************
publicsuffixlist>         Please avoid running ``setup.py`` directly.
publicsuffixlist>         Instead, use pypa/build, pypa/installer or other
publicsuffixlist>         standards-based tools.
publicsuffixlist> 
publicsuffixlist>         See https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html for details.
publicsuffixlist>         ********************************************************************************
publicsuffixlist> 
publicsuffixlist> !!
```

```suggestion
  pyproject = true;
```